cmake/platform/windows.cmake

```diff
@@ -41,9 +41,10 @@ if(BUILD_SHARED_LIBS)
   if(WIN32)
     function(add_library library)
       # Check if its an external, imported library (e.g. boost libs via cmake module definition)
-      list(FIND ARGN "IMPORTED" FIND_POS)
+      list(FIND ARGN "IMPORTED" FIND_IMPORTED)
+      list(FIND ARGN "ALIAS" FIND_ALIAS)
```

Comment:
Avoid using set_target_properties on `ALIAS` targets

This change is to fix the error like below:
```
2018-11-30T10:40:39.7610068Z CMake Error at C:/opt/ros/melodic/x64/share/catkin/cmake/platform/windows.cmake:47 (set_target_properties):
2018-11-30T10:40:39.7610963Z   set_target_properties can not be used on an ALIAS target.
2018-11-30T10:40:39.7611263Z Call Stack (most recent call first):
2018-11-30T10:40:39.7611903Z   abseil_cpp/CMake/AbseilHelpers.cmake:67 (add_library)
2018-11-30T10:40:39.7612185Z   abseil_cpp/absl/base/CMakeLists.txt:74 (absl_library)
```
https://github.com/ms-iot/catkin/commit/8545d9f28f0d5bd93ab0f6fa2348e1585a74bea7#diff-2249c319ebe44cfef5e351de7305d244